lldb/source/Interpreter/CommandReturnObject.cpp

```diff
@@ -27,6 +27,12 @@ static llvm::raw_ostream &warning(Stream &strm) {
          << "warning: ";
 }
 
+static llvm::raw_ostream &note(Stream &strm) {
```

Comment:
Since `note` is a function should it be capitalized to follow the LLDB style?

Reply:
Actually `error` and `warning` are not capitalized, I guess we can either keep all three lower case or capitalize all of them, up to you